pkgs/top-level/all-packages.nix

```diff
@@ -6899,7 +6900,7 @@ in
 
   crossLibcStdenv = overrideCC stdenv
     (if stdenv.targetPlatform.useLLVM or false
-     then buildPackages.llvmPackages_7.lldClangNoLibc
+     then buildPackages.llvmPackages_8.lldClangNoLibc
```

Comment:
Do you think a version makes sense here? Not sure how to handle it, but I would prefer if just used the "stable" llvmPackages. I guess we can do that later once everything is on llvm 8.

Reply:
That's to be as close to gcc as possible.

Reply:
(dropping this change, can revisit later as you say :))